docker-compose-2.9.yml

```diff
@@ -26,25 +26,38 @@ services:
       - ckan_storage-2.9:/var/lib/ckan
       - ./logs/2.9:/var/log/ckan
     depends_on: 
-      - db-2.9
+      - db-2.9-13
       - solr-2.9
       - redis-2.9
       - static-mock-harvest-source-2.9
     command: bash -c "/srv/app/start_ckan_development.sh"
+    # command: bash -c "tail -f /dev/null"
     networks:
       - ckan-2.9
 
-  db-2.9:
-    container_name: db-2.9
+  # db-2.9:
+  #   container_name: db-2.9
+  #   env_file:
+  #     - .env-2.9
+  #   build:
+  #     context: postgresql/
+  #   volumes:
+  #     - pg_data-2.9:/var/lib/postgresql/data
+  #   networks:
+  #     - ckan-2.9
```

Comment:
Not a blocker, but in general I wouldn't leave commented out blocks like this. If we need to compare how it was before, we have git for that.

Reply:
OK, removed the commented out code